pages/12.Plugins/20.Zapier/docs.en.md

```diff
@@ -17,21 +17,30 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
 
 ## Requirements
 
-1. Mautic version 2.9.0 or newer with SSL - basic authentication requires HTTPS to be secure.
-3. Zapier account. Free or Paid. [View Zapier pricing][zapier-pricing].
+Zapier account. Free or Paid. [View Zapier pricing](https://zapier.com/pricing/).
 
 ## Installation
 
-There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic. This Zapier integration use Basic Authentication to access Mautic API. It is disabled by default so enable this first.
+There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic.
 
-1. Open the admin menu
-2. Go to Mautic's global configuration
-3. Go to API Settings
-4. Set __API enabled?__ to __Yes__.
-5. Set __Enable HTTP basic auth?__ to __Yes__.
-6. Save the configuration.
+### Setup Oauth2 Authentication in Mautic for Zapier 3.x
 
-![How to enable API and basic auth](enable-api.png)
+This new version of the app improves authentication by implementing the Oauth2 protocol
+
+1. Open the admin menu	1. Open the admin menu
+2. Go to Mautic's global configuration	2. Go to Mautic's global configuration > API Settings
+3. Go to API Settings	3. Set __API enabled?__ to __Yes__.
+4. Set __API enabled?__ to __Yes__.	4. Save the configuration.
+5. Set __Enable HTTP basic auth?__ to __Yes__.	5. Go to Mautic's global configuration > API Credentials > __Oauth2__
+6. Save the configuration.	6. Create new __Oauth2___ application with redirect URI: __tba__
+7. Copy Public Key and Secret Key of your created app
+
+![How to enable the API](https://user-images.githubusercontent.com/462477/74520415-cc616b00-4f17-11ea-8dbe-f6dd8a0a6cfc.png)
+
+
+![How to enable API and basic auth](https://www.mautic.org/wp-content/uploads/2018/02/enable-api.png)	__Note__: **Refresh token lifetime (in days)**  means how many days need app to refresh token. If your app will not have any activity in that period, you have to re-authorize it again. 
```

Comment:
Let's make the note look pretty 🤩  Reference: https://demo.hibbittsdesign.org/grav-learn2-git-sync/intermediate/overview#blue

```suggestion
![How to enable API and basic auth](https://www.mautic.org/wp-content/uploads/2018/02/enable-api.png)	

>>>>>**Refresh token lifetime (in days)**  means how many days needed before the app needs to refresh the token. If your app does not have any activity in that period, you have to re-authorize it again. 
```